kover-gradle-plugin/docs/index.md

```diff
@@ -85,223 +757,374 @@ buildscript {
 apply plugin: 'org.jetbrains.kotlinx.kover'    
 ```
 
-## Using Kover tasks
-Using JVM projects and Android projects differs, use the recommendations below according to your project type.
+#### Application in submodule with Plugins DSL
 
-### Kotlin JVM project
-These are projects that use Kotlin Gradle plugin.
+If you are using a plugin in a multi-module build, then there is no need to re-specify the plugin version in the submodules, 
+and it is enough to write like this:
+```kotlin
+plugins {
+     id("org.jetbrains.kotlinx.kover")
+}
+```
 
-For such projects, only the [Kover default tasks](#kover-default-tasks) for generating reports are used: `koverHtmlReport`, `koverXmlReport`, `koverVerify`.
+### Specify verification rules for total variant
 
-To configure reports, the [default reports settings block](configuring#configuring-default-reports) is used:
 ```kotlin
-koverReport {
-    filters {
-        // filters for all reports
-    }
-  
-    verify {
-        // verification rules for all reports
+kover {
+    reports {
+        total {
+            verify {
+                // verification rules
+            }
+        }
     }
+}
+```
+
+### Specify verification rules for named variant
 
-    defaults {
-        xml { /* default XML report config */ }
-        html { /* default HTML report config */ }
-        verify { /* default verification config */ }
-        log { /* default logging config */ }
+Configuring verification rules for named report variant `foo`
+```kotlin
+kover {
+    reports {
+        variant("foo") {
+            verify {
+                // verification rules
+            }
+        }
     }
 }
 ```
 
-### Kotlin Android project
-These are projects that use Kotlin Android Gradle plugin.
+You can read more about creating verification rules in the [corresponding section](#verification).
 
-In such projects, a [variant of Kover tasks](#kover-android-tasks) is created for each Android build variant, 
-e.g. `koverHtmlReportRelease`, `koverXmlReportRelease`, `koverVerifyRelease` for `release` build variant.
+### Configuring report tasks
```

Comment:
This section lacks some text, as it is just a showcase of DSL. You should add a little bit of 'water' to it.